README.md

```diff
@@ -1,29 +1,29 @@
 # NodeMCU 3.0.0
+> Lua-based firmware for ESP8266 WiFi SOC
```

Comment:
Having a description directly after the heading followed by badges is a common pattern on repos.